dev-manuals/test-plans/minigames/crosswordpuzzle-frontend/plans/u-crosswordpuzzle-4-restart-game.md

```diff
@@ -0,0 +1,38 @@
+# Crosswordpuzzle restart game (`u.crosswordpuzzle-4`)
+
+Version: v1.0, 2022-08-12, created test plan \
+Author: Aaron Schmid
+
+
+## Description
+
+This use case verifys that the crosswordpuzzle can be restarted.
+
+## Precondition
+
+A configuration has been stored in the backend and selected in the frontend.
```

Comment:
You can start after a successful round (prior to submitting the solution?) to omit 3/4 of the typical procedure.